### PR TITLE
s/rmconstref/decay/g

### PR DESCRIPTION
**Pull request by dkorolev**
Using a more standard term: type decay.
`make indent` has been run.

### CODE REVIEW

File: Bricks/template/decay.h [hunk 1]
```diff
@@ -134,72 +136,68 @@ static_assert(is_same<const int, rmref<const int&&>>::value, "");
...
-    "");
+    is_same<std::tuple<std::tuple<std::string>>, decay<std::tuple<std::tuple<const std::string>>>>::value, "");
+static_assert(is_same<std::tuple<std::tuple<std::string>>, decay<std::tuple<std::tuple<std::string&>>>>::value,
+              "");
 static_assert(
-    is_same<std::tuple<std::tuple<std::string>>, rmconstref<std::tuple<std::tuple<std::string&>>>>::value, "");
+    is_same<std::tuple<std::tuple<std::string>>, decay<std::tuple<std::tuple<const std::string&>>>>::value, "");
+static_assert(is_same<std::tuple<std::tuple<std::string>>, decay<std::tuple<std::tuple<std::string&&>>>>::value,
+              "");
+
```
Comment (mzhurovich) [inline Bricks/template/decay.h]: Intended linebreak?

Comment (dkorolev) [inline Bricks/template/decay.h]: Nah, just checked -- `clang-format`-s.

Comment (mzhurovich) [inline Bricks/template/decay.h]: WTF?! I thought, it supposed to format strings, but not add additional vertical whitespaces.

Comment (dkorolev) [inline Bricks/template/decay.h]: It moved `is_same` onto the same line as `static_assert`.Yeah, I know, самодеятельность и все такое. But we still love him.

Comment (mzhurovich) [inline Bricks/template/decay.h]: I see, but line 179 became just empty, right?

Comment (dkorolev) [inline Bricks/template/decay.h]: Fixed in 720c2cf

**Comment by mzhurovich** [comment]
@dkorolev LGTM!

**Comment by dkorolev** [comment]
@mzhurovich Thank you!!!

**Pull request merged**